### PR TITLE
adds additional docs to rails cop

### PR DESCRIPTION
**Pull request by logicminds**
This commit adds some more documentation inside the cop to alert the user that this cop is a rails5 only cop.

-----------------

Before submitting the PR make sure the following are checked:

* [ x ] Wrote [good commit messages][1].
* [ x ] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [ x ] Used the same coding conventions as the rest of the project.
* [ x ] Feature branch is up-to-date with `master` (if not - rebase it).
* [ x ] Squashed related commits together.
* [ x ] Added tests.
* [ x ] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [ x ] All tests are passing.
* [ x  ] The new code doesn't generate RuboCop offenses.
* [ x ] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [ x ] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

### CODE REVIEW

File: lib/rubocop/cop/rails/http_positional_arguments.rb [hunk 1]
```diff
@@ -14,7 +16,7 @@ module Rails
       #   # good
       #   get :new, params: { user_id: 1 }
       class HttpPositionalArguments < Cop
-        MSG = 'Use keyword arguments instead of ' \
+        MSG = 'Rails5: Use keyword arguments instead of ' \
```
Comment (bbatsov) [inline lib/rubocop/cop/rails/http_positional_arguments.rb]: Please remove this change. I don't like have version specific cop messages.

**Comment by bbatsov** [comment]
Btw, you didn't re-run the manual creation task after changing this cop description. And for some reason `util.rb` is part of your changeset.

**Comment by logicminds** [comment]
Manual creation task?

**Comment by bbatsov** [comment]
> [ x ] Updated cop documentation with rake generate_cops_documentation (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

Guess you didn't read carefully this part. :-)

**Comment by logicminds** [comment]
Just ran the rake task.  I didn't know I needed to run the task since I thought it applied to updating other documentation that was part of the code.

**Pull request merged**